.env

```diff
@@ -196,3 +196,20 @@ MF_POSTGRES_READER_DB_SSL_MODE=disable
 MF_POSTGRES_READER_DB_SSL_CERT=""
 MF_POSTGRES_READER_DB_SSL_KEY=""
 MF_POSTGRES_READER_DB_SSL_ROOT_CERT=""
+
+# Twins
+MF_TWINS_LOG_LEVEL=debug
+MF_TWINS_HTTP_PORT=9021
+MF_TWINS_SERVER_CERT=""
+MF_TWINS_SERVER_KEY=""
+MF_TWINS_DB_NAME=mainflux
+MF_TWINS_DB_HOST=twins-db
+MF_TWINS_DB_PORT=27017
+MF_TWINS_SINGLE_USER_EMAIL=""
+MF_TWINS_SINGLE_USER_TOKEN=""
+MF_TWINS_CLIENT_TLS=""
+MF_TWINS_CA_CERTS=""
+MF_TWINS_MQTT_URL=tcp://mqtt-adapter:1883
+MF_TWINS_THING_ID=cfc93379-d09a-438b-806e-e3cda2180124
```

Comment:
Do we need these default values, or is it better to leave this empty?

Reply:
Unfotunatelly, these values are necessary. Therefore, we should leave them empty. What is more, I will move twins service to addons.

Reply:
I'm working towards eliminating these env vars alltogether.

Reply:
some vars are not used in docker-compose

Reply:
Is this still ongoing?